backend/jobs/jiraJob.ts

```diff
@@ -118,33 +39,61 @@ async function fetchSprints(
           jiraIssues: [],
         };
 
-        const jiraSprintId = await findJiraSprintId(jiraSprint.id, cloudId);
-
-        let sprint;
-        if (jiraSprintId) {
-          sprint = await JiraSprintModel.findOneAndUpdate(
-            { _id: jiraSprintId },
-            jiraSprint,
-            {
-              upsert: true,
-              new: true,
-            }
-          );
-        } else {
-          sprint = await new JiraSprintModel(jiraSprint).save();
-        }
+        const self = jiraSprint.self;
+        const sprint = await JiraSprintModel.findOneAndUpdate(
+          { self: self },
+          jiraSprint,
+          {
+            upsert: true,
+            new: true,
+          }
+        );
 
-        const jiraBoardId = await findJiraBoardId(boardId, cloudId);
+        const boardSelfUri = `https://api.atlassian.com/ex/jira/${cloudId}/rest/agile/1.0/board/${boardId}`;
```

Comment:
Can consider refactoring all the fixed endpoints to a separate `constants.ts` file for better visibility

Reply:
Created a `endpoints.ts` file in utils folder for the Atlassian and Jira urls.